### PR TITLE
NewsWeek page clone

### PR DESCRIPTION
**Pull request by uryelah**
Semantic Markup

Grid system with Bootstrap

Responsive grid

Breakpoints for small devices

Styling of the page based on the original Newsweek layout

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1947 @@
...
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://kit.fontawesome.com/d8dd4cbcc4.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="normalize.css" />
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (bolah2009) [inline index.html]: Let's use the [Responsive meta tag](https://getbootstrap.com/docs/4.3/getting-started/introduction/#responsive-meta-tag)

```html
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+.txt-red {
+    color: var(--news-red)
+}
+
+.max-1290 {
+    max-width: 1290px;
+}
+
+section.max-1290 {
+    max-width: 1600px !important;
```
Comment (bolah2009) [inline style.css]: #### `REQUIRED`

Always look for a way to use [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) before even considering `!important`. 

Learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception)

> Make sure you DON’T make [common mistakes](https://speckyboy.com/good-bad-css-practices/) in your CSS styles.

> **Recommendation:** You should only use the !important rule when overriding styles in a user style sheet, overriding 3rd party code and inline styles as well as utility classes.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+}
+
+.bg-secondary {
+    background-color: #f4f5f6 !important;
```
Comment (bolah2009) [inline style.css]: Always look for a way to use [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) before even considering `!important`.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+/*==========================================================
+    MEDIA QUERIES
+==========================================================*/
+@media only screen and (min-width: 760px) {
+    main:first-of-type {
+        height: 2150px;
+    }
+    
+    main #main-section {
+        height: 2020px !important;
```
Comment (bolah2009) [inline style.css]: Does deleting this `!important` affects this element? :thinking:

File: style.css [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+}
+/*==========================================================
+    GENERAL TAGS
+==========================================================*/
+body {
+    font-family: Arial,Helvetica,sans-serif;
+}
+
+.hid {
+    display: none;
```
Comment (bolah2009) [inline style.css]: Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

Bootstrap class names can be used instead. See [Display](https://getbootstrap.com/docs/4.1/utilities/display/) in bootstrap documentation.

> Remember to use Bootstrap CSS classes wherever it is possible - this is the main point of this project.

#### Please, lets do this for all other occurrences. :blush:

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Nice job @uryelah 

## Good job with [the clone](https://rawcdn.githack.com/uryelah/Using-Bootstrap/40542364422b366f1d767da61ab8882a95a232a0/index.html), it really looks nice and uses bootstrap effectively. 

### Please, check out the comment below and submit another CRR.

Also, why not hide the nav links in in larger screen, currently, the nav links is truncated for the smaller screen. See source page breakpoints for the nav link for more details. 

#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

File: style.css (hunk 5 above)
Comment (uryelah) [reply style.css]: Done!

File: index.html (hunk 1 above)
Comment (uryelah) [reply index.html]: Nice, added.

File: style.css (hunk 4 above)
Comment (uryelah) [reply style.css]: Nope it does not, deleted.
>

File: style.css (hunk 3 above)
Comment (uryelah) [reply style.css]: Done.

File: style.css (hunk 2 above)
Comment (uryelah) [reply style.css]: Done. The important tag ended up not being needed, removed.

**Comment by ebukaume** [comment]
Hi @uryelah :wave: 

Nice job implementing those requested changes. I would have loved to give you a green but unfortunately, there is a validation error in your HTML. Please always try to validate your codes first before requesting a review; that way it says you some time on that front.

Kindly run your code through [this free online validator](https://validator.w3.org/nu/) and fix the error [here](https://github.com/uryelah/Using-Bootstrap/blob/news/index.html#L24).

[OPTIONAL]
While at it, you can make your README to be more descriptive and look professional. You can refer to [this](https://github.com/othneildrew/Best-README-Template) for a little inspiration.

**Please submit another code review request once you have work on those.**

Happy Coding :v: 
**Ebuka Umeokonkwo**
[Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Twitter](https://twitter.com/ebukaume)

**Comment by ebukaume** [comment]
Hi @uryelah :wave:

**Congratulations, you can now merge this branch.**

I wish you all the best in your next project.

Happy Coding :v: 
**Ebuka Umeokonkwo**
[Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Twitter](https://twitter.com/ebukaume)

**Pull request merged**